tutorials/3d/procedural_geometry/introduction.rst

```diff
@@ -0,0 +1,106 @@
+.. _doc_procedural_geometry_introduction:
+
+Introduction to procedural geometry
+===================================
+
+There are many ways to procedurally generate geometry in Godot. In this tutorial series
+we will explore a few of them. Each technique has its own benefits and drawbacks, so 
+it is best to understand each one and how it can be useful in a given situation. 
+
+What is geometry?
+-----------------
+
+Geometry is a fancy way of saying shape. In computer graphics, geometry is typically represented
+by an array of positions called "vertices". In Godot, geometry is represented by Meshes.
```

Comment:
Isn't a mesh also composed of vertices?

Reply:
Kind of. Geometry is a math term for a 3D object while in graphics you usually represent geometry using a mesh. Which is an object with vertices, materials etc. In Godot it is slightly more complicated as a mesh is really just a container for surfaces, as explained below. So even though i have said geometry is represented by meshes, it would be inaccurate to say meshes are composed of vertices in godot. As meshes are a higher-level construct